doc/nrf/protocols/lte/psm.rst

```diff
@@ -0,0 +1,267 @@
+.. _cellular_psm:
+
+Power saving techniques
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Cellular technology is an example of an LPWAN (Low Power Wide Area Network) because to its coverage, high mobility, roaming capabilities, and infrastructure.
+However, battery lifespan can be a problem for devices using cellular IoT if you do not fully understand the power-saving capabilities and how to use them to decrease power consumption.
+
+This guide explains following concepts for saving power:
+
+* :ref:`Radio activity modes <cellular_radio_modes>`, to enable you to examine power-saving modes in cellular IoT.
+* :ref:`cellular_power_saving`:
+
+  * Discontinuous Reception
+  * Connected mode DRX (cDRX)
+  * Idle-mode DRX (iDRX)
+  * extended Discontinuous Reception (eDRX)
+  * Power Saving Mode (PSM)
+  * Reduced mobility
+  * Country-specific search optimization
+  * Abort network search early
+  * Connection evaluation or Energy estimation
```

Comment:
change name to: 
Connection Quality Indicator (CQI)

Reply:
Both the modem AT command documentation and LTE Link Control API use the term "energy estimate". Is the CQI term used already somewhere?

Reply:
Updated the section as ""Connection Quality Indicator (CQI) or Energy estimation.